backend/devices/bitbox/relay/channel.go

```diff
@@ -267,13 +268,13 @@ func (channel *Channel) SendXpubEcho(xpubEcho string, typ string) error {
 // SendSigningEcho sends the encrypted signing echo from the BitBox to the paired mobile.
 func (channel *Channel) SendSigningEcho(
 	signingEcho string,
-	coin string,
+	coin coin.Code,
 	scriptType string,
 	transaction string,
 ) error {
 	return PushMessage(relayServer(), channel, map[string]string{
 		"echo":               signingEcho,
-		"coin":               coin,
+		"coin":               string(coin),
```

Comment:
maybe this should be an explicit map, even though key and value would be the same (in theory one should be able to change the coin codes and everything should work fine after, right?).

Reply:
I would leave it as is. How frequent do the codes change? If you rename a coin code in Go but keep it unchanged in the frontend, I'd argue it might become confusing. So, if I needed to rename or add a new code, I'd prefer to modify both backend and the frontend.